atc/routes.go

```diff
@@ -161,6 +163,9 @@ var Routes = rata.Routes([]rata.Route{
 	{Path: "/api/v1/teams/:team_name/pipelines/:pipeline_name/resources", Method: "GET", Name: ListResources},
 	{Path: "/api/v1/teams/:team_name/pipelines/:pipeline_name/resource-types", Method: "GET", Name: ListResourceTypes},
 	{Path: "/api/v1/teams/:team_name/pipelines/:pipeline_name/resources/:resource_name", Method: "GET", Name: GetResource},
+	{Path: "/api/v1/teams/:team_name/pipelines/:pipeline_name/resources/:resource_name/cache", Method: "DELETE", Name: ClearResourceCaches},
```

Comment:
One thing to keep in mind might be that in order to clear a resource cache, it has to not be used by anything. If we run a command to clear all the resource caches for a resource, that most likely will end up erroring because something is using a cache (for example, a build is currently running and using a cache that we are trying to delete).

I think it would be fine to error and have the fly command tell the user that something is currently using a cache that it is trying to delete, since we shouldn't be pulling out caches underneath builds that are using it. 🤔